src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/CharpSyntaxFormattingOptions.cs

```diff
@@ -0,0 +1,280 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
nit: File name has a typo (`Charp` -> `CSharp`)

Reply:
@tmat It seems this was marked as resolved but is still an issue (it's not a big deal though, but can be annoying when searching by file name in GitHub or VS)

Reply:
Oops. Something went wrong in my VS. Should be fixed now.